### ISSUE TITLE
Overview | PM2 Documentation

### ISSUE THREAD

**Issue by robin-monnier** [keymetrics/doc-pm2#168]
https://pm2.io/doc/en/runtime/overview/ 

 PM2 - Guide, References, Tutorials and Best Practices about PM2 and Node.js.

**Comment by waoowa827** [comment]
can i use it instead of tmux?

**Comment by afkin9** [comment]
> @waoowa827
> can i use it instead of tmux?

Yes, PM2 is much better for running your node apps :)

**Comment by ranwawa** [comment]
英文能力需要提升呀
这个评论插件是github提供的么
is  this comment plugin  came from gitgub?

**Comment by SURA907** [comment]
- can i hope the pm2 can manage https ca certificate
> it means my app is run on http
> i have a ca cert (but i don't hope change the app)
> i hope pm2 can use the ca cert to make my app run https

**Comment by cmforjava** [comment]
I just run 'pm2 serve', but the popups (DOS) keep disturbing me

**Comment by sentax** [comment]
> @cmforjava
> I just run 'pm2 serve', but the popups (DOS) keep disturbing me

Change your "serve" script from package.json file to =>

"serve": "cross-env NODE_ENV=production pm2 start ./server/index.js"

**Comment by brizer** [comment]
The gitalk is amazing

**Comment by kumarshambhu** [comment]
Can we use PM2 runtime, free in production server, without any restriction?
what all are restriction ?

**Comment by TheGrimSilence** [comment]
This is beautiful

**Comment by TheGrimSilence** [comment]
> @kumarshambhu
> Can we use PM2 runtime, free in production server, without any restriction?
> what all are restriction ?

Yes you can, there are no restrictions from what I see, it's open-source.

**Comment by kumarshambhu** [comment]
It is opensource but AGPL license.
